### PR TITLE
auth docs: shift EOL generations

### PR DESCRIPTION
**Pull request by Habbie**
### Short description
4.3 is out, so 4.0 is EOL.

### Checklist

I have:
- [ ] read the [CONTRIBUTING.md](https://github.com/PowerDNS/pdns/blob/master/CONTRIBUTING.md) document
- [ ] compiled this code
- [ ] tested this code
- [ ] included documentation (including possible behaviour changes)
- [ ] documented the code
- [ ] added or modified regression test(s)
- [ ] added or modified unit test(s)
- [ ]  checked that this code was merged to master

### CODE REVIEW

**Comment by zeha** [comment]
maybe this should say something about 4.0 too.

File: docs/appendices/EOL.rst [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
-PowerDNS Authoritative Server 4.1 will only receive correctness, stability and security updates and will be receiving security updates only after PowerDNS Authoritative Server 4.3 is released.
-It wil be end of life after PowerDNS Authoritative Server 4.4 is released.
+PowerDNS Authoritative Server 4.2 will only receive correctness, stability and security updates and will be receiving security updates only after PowerDNS Authoritative Server 4.4 is released.
+It wil be end of life after PowerDNS Authoritative Server 4.5 is released.
 
-PowerDNS Authoritative Server 4.0 will only receive security updates and will be end of life after PowerDNS Authoritative Server 4.3 is released.
+PowerDNS Authoritative Server 4.1 will only receive security updates and will be end of life after PowerDNS Authoritative Server 4.4 is released.
 
 PowerDNS Authoritative Server 3.x and 2.x are end of life, and will not
```
Comment (omoerbeek) [inline docs/appendices/EOL.rst:11]:
```suggestion
PowerDNS Authoritative Server 4.0, 3.x and 2.x are end of life, and will not
```

File: docs/appendices/EOL.rst [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-The currently supported release train of PowerDNS Authoritative Server is 4.2.
+The currently supported release train of PowerDNS Authoritative Server is 4.3.
 
-PowerDNS Authoritative Server 4.1 will only receive correctness, stability and security updates and will be receiving security updates only after PowerDNS Authoritative Server 4.3 is released.
-It wil be end of life after PowerDNS Authoritative Server 4.4 is released.
+PowerDNS Authoritative Server 4.2 will only receive correctness, stability and security updates and will be receiving security updates only after PowerDNS Authoritative Server 4.4 is released.
+It wil be end of life after PowerDNS Authoritative Server 4.5 is released.
 
-PowerDNS Authoritative Server 4.0 will only receive security updates and will be end of life after PowerDNS Authoritative Server 4.3 is released.
+PowerDNS Authoritative Server 4.1 will only receive security updates and will be end of life after PowerDNS Authoritative Server 4.4 is released.
```
Comment (omoerbeek) [inline docs/appendices/EOL.rst:9]:
```suggestion
PowerDNS Authoritative Server 4.1 will only receive critical security updates and will be end of life after PowerDNS Authoritative Server 4.4 is released.
```

**Comment by Habbie** [comment]
thanks both, amended

**Comment by zeha** [comment]
further down it says `All users are urged to upgrade to version 4.1` ...

**Comment by Habbie** [comment]
> further down it says `All users are urged to upgrade to version 4.1` ...

ah.. not wrong, but not great. Also amended now :)

**Pull request merged**